packages/db/prisma/schema.prisma

```diff
@@ -68,13 +68,13 @@ enum Gender {
 }
 
```

Comment:
can't seed data anymore

Reply:
Oh boi, we gonna have a lot of talk for this schema change🔥🔥🔥

Reply:
okay gotcha